src/pudl_archiver/metadata/sources.py

```diff
@@ -21,8 +21,81 @@
+        "path": "https://www.energy.gov/scep/low-income-energy-affordability-data-lead-tool",
+        "description": (
+            "This archive includes the data behind the Department of Energy's (DOE)"
+            "Low Income Energy Affordability Data (LEAD) tool. The LEAD tool is an"
+            "online, interactive platform that helps users make data-driven decisions"
+            "on energy goals and program planning by improving their understanding of"
+            "low-income and moderate-income household energy characteristics. The LEAD"
+            "Tool offers the ability to select and combine geographic areas (state,"
+            "county, city and census tract) into one customized group so users can see"
+            "the total area for their customized geographies (e.g., specific service"
+            "territories)."
+        ),
+        "working_partitions": {"years": [2022, 2018]},
+        "keywords": sorted(
+            {"doe", "lead", "low income", "energy affordability", "energy burden"}
+        ),
+        "license_raw": LICENSES["us-govt"],
+        "license_pudl": LICENSES["cc-by-4.0"],
+        "contributors": [CONTRIBUTORS["catalyst-cooperative"]],
+    },
+    "eiacbecs": {
+        "title": "EIA CBECS -- Commercial Buildings Energy Consumption Survey",
+        "path": "https://www.eia.gov/consumption/manufacturing/data/2018/",
+        "description": (
+            "The Commercial Buildings Energy Consumption Survey (CBECS) is a national"
+            "sample survey that collects information on the stock of U.S. commercial"
+            "buildings, including their energy-related building characteristics and"
+            "energy usage data (consumption and expenditures). Commercial buildings"
+            "include all buildings in which at least half of the floorspace is used for"
+            "a purpose that is not residential, industrial, or agricultural. By this"
+            "definition, CBECS includes building types that might not traditionally be"
+            "considered commercial, such as schools, hospitals, correctional"
+            "institutions, and buildings used for religious worship, in addition to"
+            "traditional commercial buildings such as stores, restaurants, warehouses,"
+            "and office buildings."
+        ),
+        "working_partitions": {
+            "years": [
+                2003,
+                2012,
+                2018,
+            ]  # there are PDF only versions for 1999, 1995 and 1992
+        },
+        "keywords": sorted(
+            {"eia", "energy", "cbecs", "consumption", "buildings", "commercial"}
+        ),
+        "license_raw": LICENSES["us-govt"],
+        "license_pudl": LICENSES["cc-by-4.0"],
+        "contributors": [CONTRIBUTORS["catalyst-cooperative"]],
+    },
+    "eianems": {
+        "title": "EIA NEMS -- National Energy Modeling System",
+        "path": "https://github.com/EIAgov/NEMS/tree/main?tab=readme-ov-file",
```

Comment:
Point to the canonical repository URL.

```suggestion
        "path": "https://github.com/EIAgov/NEMS",
```